src/constants/commonOptions.js

```diff
@@ -19,6 +19,11 @@ module.exports = {
     placeholder: 'Placeholder',
     tooltip: 'The placeholder text that will appear when this field is empty.'
   },
+  description: {
```

Comment:
Don't you also want to add this to the settings of the component?  I like what you have added here. Thanks for your contribution.